tests/superphot_plus/data_generation/test_make_fake_spp_data.py

```diff
@@ -14,6 +14,14 @@ def test_generate_clean_data():
         assert lcs[i].shape == (4, 50)
         assert len(params[i]) == 14
 
+    # Generate 5 light curves with 10 time steps each and 3 bands
+    params, lcs = create_clean_models(5, 10, bands=["r", "g", "i"])
```

Comment:
... and then check here that you're getting lightcurve data in all three bands.

Reply:
Done